src/pathfind/teleport.hpp

```diff
@@ -18,6 +18,8 @@
 #include "config.hpp"
 #include "map/location.hpp"
 
+#include <unordered_set>
```

Comment:
Looks like an include for `unordered_map` is missing.

Reply:
I added it.